drivers/hmis/app/graphql/schema.graphql

```diff
@@ -1598,6 +1598,52 @@ type CreateScanCardCodePayload {
   scanCardCode: ScanCardCode
 }
 
+"""
+Autogenerated input type of CreateServiceCategory
+"""
+input CreateServiceCategoryInput {
+  """
+  A unique identifier for the client performing the mutation.
+  """
+  clientMutationId: String
+  name: String!
+}
+
+"""
+Autogenerated return type of CreateServiceCategory.
+"""
+type CreateServiceCategoryPayload {
+  """
+  A unique identifier for the client performing the mutation.
+  """
+  clientMutationId: String
+  errors: [ValidationError!]!
+  serviceCategory: ServiceCategory
+}
+
+"""
+Autogenerated input type of CreateServiceType
+"""
+input CreateServiceTypeInput {
```

Comment:
Switching to CleanBaseMutation should get rid of these noisy input wrapper types I think